backend/clickhouse/logs.go

```diff
@@ -166,6 +166,59 @@ func (client *Client) ReadLogs(ctx context.Context, projectID int, params modelI
 	return getLogsConnection(edges, pagination), rows.Err()
 }
 
+// This is a lighter weight version of the previous function for loading the minimal about of data for a session
+func (client *Client) ReadSessionLogs(ctx context.Context, projectID int, params modelInputs.LogsParamsInput) ([]*modelInputs.LogEdge, error) {
```

Comment:
The frontend is still responsible for constructing the query (e.g. `secure_session_id:foo`) ([example](https://github.com/highlight/highlight/pull/4833/files#diff-0f0e22bb410b4636bf449d61d3dc5d7faba340c3198f7baa2b284683774e7896R91)) because it'll need to do that anyways to construct the query for when we click the "Show in Log Viewer" button (see #4779 for requirements)